ray-operator/config/crd/bases/ray.io_rayclusters.yaml

```diff
@@ -34,6 +34,51 @@ spec:
           spec:
             description: Specification of the desired behavior of the RayCluster.
             properties:
+              autoscalerOptions:
+                description: AutoscalerOptions specifies optional configuration for
+                  the Ray autoscaler.
+                properties:
+                  idleTimeoutSeconds:
+                    description: IdleTimeoutSeconds is the number of seconds to wait
+                      before scaling down a worker pod which is not us
+                    format: int32
+                    type: integer
+                  image:
+                    description: Image optionally overrides the autoscaler's container
+                      image.
+                    type: string
+                  resources:
+                    description: Resources specifies resource requests and limits
+                      for the autoscaler container.
+                    properties:
+                      limits:
+                        additionalProperties:
+                          anyOf:
+                          - type: integer
+                          - type: string
+                          pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
```

Comment:
Could you add an explanation for this pattern? Same for line 69

Reply:
Ah, these are the auto-generated internals of a K8s container's resource field.

The readable source of truth is in `raycluster_types.go`.

Reply:
The pattern is a regex encoding the rules for specifying resource quantities, as described in the K8s docs:
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes